cmd/admin-config-export.go

```diff
@@ -46,13 +45,13 @@ EXAMPLES:
 
 // configExportMessage container to hold locks information.
 type configExportMessage struct {
-	Status string `json:"status"`
-	Value  []byte `json:"value"`
+	Status string      `json:"status"`
+	Value  interface{} `json:"value"`
 }
 
 // String colorized service status message.
 func (u configExportMessage) String() string {
-	return string(u.Value)
+	return u.Value.(string)
```

Comment:
You need to revert this change as well @wlan0 - this will crash

Reply:
done